vm-migration/src/lib.rs

```diff
@@ -279,4 +289,16 @@ pub trait Transportable: Pausable + Snapshottable {
 /// and Snapshottable.
 /// Moreover a migratable component can be transported to a remote or local
 /// destination and thus must be Transportable.
-pub trait Migratable: Send + Pausable + Snapshottable + Transportable {}
+pub trait Migratable: Send + Pausable + Snapshottable + Transportable {
+    fn start_migration(&mut self) -> std::result::Result<(), MigratableError> {
+        Ok(())
+    }
+
+    fn stop_migration(&mut self) -> std::result::Result<(), MigratableError> {
+        Ok(())
+    }
```

Comment:
Considering what we do with `{start, stop}_migration()` now are all about dirty logging, do you think it would be better to name the trait APIs as `{start, stop}_dirty_log()`?

In this way, I think it would read (a bit) better with our current migration protocol/workflow. E.g. 1) It will not confuse with the "vm_migration::protocol::Request::{Start, Complete}" steps and related comments; 2) it will explains itself why  "{start, stop}_migration()" is needed only when sending the (from the source vm) and is not needed when receiving.